src/integrationTest/stepFunctions/visualizeStateMachine.test.ts

```diff
@@ -83,6 +85,12 @@ async function waitUntilWebviewIsVisible(webviewPanel: vscode.WebviewPanel | und
 }
 
 describe('visualizeStateMachine', async () => {
+    before(async function() {
+        // tslint:disable-next-line:no-invalid-this
+        this.timeout(600000)
```

Comment:
What's our individual test timeout? I'm surprised this works.

Reply:
Default timeouts are 2 seconds. The integration tests (defined as "using a running copy of the extension") need to activate the extension first.